web/layout.html

```diff
@@ -6,8 +6,22 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{.Title}}</title>
     <style>{{.Style}}</style>
+    <script>{{.Js}}</script>
 </head>
 
+{{if .IsWalletInitialised}}
+<body onload="postPass()">
+<section class="section">
+    <div class="container">
+        <h1 class="title is-1">🔌TDEX Connect URL</h1>
+        <div class="box">
+            <strong>URL:</strong>
+            <div class="content" id="url">
+            </div>
+        </div>
+    </div>
+</section>
+{{else}}
```

Comment:
not clear why this is needed: we always inject a URL, page not care if is complete (with macaroon) or not

Reply:
If wallet is not initialised we dont want to prompt user for pass.